ci/workflows/pull-request.yml

```diff
@@ -161,3 +161,30 @@ steps:
           files: ./**/*.sh
     timeout_in_minutes: 5
     retry: *auto-retry
+
+  - label: "watch sqlsmith changes"
```

Comment:
Why add this conditional step? The sqlsmith test is included in the unit-test. If you modify these two files, the unit-test of the pull-request pipeline will include the check of sqlsmith.

Reply:
> The sqlsmith test is included in the unit-test.
This PR separates it from unit-test. See: https://github.com/singularity-data/risingwave/pull/4134/files?diff=unified&w=0#diff-9d4646f9b17a55d03b48a18999d4f9deaf963950e62ec8870d0d15e21562d39dR15.

Reply:
I'm not against splitting it out in the cron pipeline, I don't think adding a conditional step to the pull-request pipeline is necessary. Because unit-test in the pull-request pipeline includes sqlsmith check.

Reply:
The conditional check is needed to prevent normal PR workflows from running sqlsmith in unit tests.

Reply:
OK，thanks for clarifying things👍